src/app/static/src/scss/partials/maps.scss

```diff
@@ -7,6 +7,15 @@
   display: table-cell;
 }
 
+.emptyFeature-center {
```

Comment:
Sorry I'm going to be a complete pedant and ask you to change this to kebab-case like the other css classes (`empty-feature-center`)!